src/styles/typography/index.md

```diff
@@ -157,6 +157,18 @@ For example, links in a header or side navigation might not need underlines. Use
 
 {{ example({group: "styles", item: "typography", example: "link-no-underline", html: true, open: true}) }}
 
+### Links to change a language
+
+Links can be used to allow a user to access the current content in a different language.
+
+When offering links to content in other languages, make sure:
+
+- the link's content includes the language name in both English and the source language
```

Comment:
I think we may also want to specify that the name of the source language warrants a `lang` attribute of its own, as it is itself in a different language. 
 
That does make me wonder if this is getting so in-the-weeds that a coded example might make it easier to understand the bullets? For example: 

```html
You can also use <a href="..." hreflang="fr">the tax filing service in French (<span lang="fr">français</span>)</a>.
```

Reply:
I like this idea, maybe we could do that to the written example later on

Reply:
I was thinking this might be more of a stopgap before we make a language link component. There are a bunch of those types of considerations, for sure.

This added content doesn't even give a specific example of the text to put in the link / paragraph at the moment (unless @CharlotteDowns is referring to a coded example we're planning to add which would include that suggested language?)

Reply:
In my mind this is very much a stopgap until we have more in-depth guidance, and potentially a new pattern or component.

My previous work on localisation revealed a whole mess of different language switcher interfaces currently in use. This guidance update only really covers the 'indent text' styled version, which is one of the more common ones. It's still an update worth having, but it definitely doesn't cover all of the existing bases. 

![A collage of screenshots showing language switchers from various GOV.UK pages and services, including inline and stacked lists of languages, toggle switches, and permanent links in the header or footer; some include the language name in English whilst others don't.](https://github.com/alphagov/govuk-design-system/assets/1253214/1ed99014-90ba-434c-8417-36c7945d77b5)